3-enrich/stream-enrich/examples/config.hocon.sample

```diff
@@ -92,7 +94,25 @@ enrich {
       retries = 0
     }
 
+    # config for nsq
+    nsq {
+      # Channel name for nsq source
+      # If more than one application reading from the same NSQ topic at the same time,
+      # all of them must have unique channel name for getting all the data from the same topic
+      rawChannel: "{{nsqSourceChannelName}}"
+
+      # Host name for NSQ tools
+      host: "{{nsqHost}}"
+
+      # HTTP port for nsqd
+      port: "{{nsqdPort}}"
+
+      # HTTP port for nsqlookupd
+      lookupPort: {{nsqlookupdPort}}
+    }
+
     # After enrichment, events are accumulated in a buffer before being sent to Kinesis/Kafka.
+    # Note: Buffering is not supported by NSQ.
```

Comment:
can't we use the recordLimit like in the loaders?

Reply:
Sure we can I just did not see it necessary because they can handle fairly good amount of request and snowplow mini is real time, buffering in this tools would break being real time. However, if you want, I can add the buffering.

Reply:
true that in the case of NSQ, we're only buffering incoming events, let's leave it as is